go.mod

```diff
@@ -59,4 +59,6 @@ replace (
 
 replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309 // Required by Helm
 
-replace github.com/openshift/api => github.com/openshift/api v0.0.0-20190924102528-32369d4db2ad // Required until https://github.com/operator-framework/operator-lifecycle-manager/pull/1241 is resolved
+replace github.com/openshift/api => github.com/openshift/api v0.0.0-20200527184302-a843dc3262a0 // Required until https://github.com/operator-framework/operator-lifecycle-manager/pull/1241 is resolved
+
+replace github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20191125132246-f6563a70e19a
```

Comment:
Is this a hard requirement to manage console link resources? current openshift go client version does not support consolelinks?

Reply:
Consolelink exists in this existing version, but it will need further investigation to confirm if it works just like the one in the newer version in this new client-go.
However, the operator-util we used to verify consolelink before add/update it does request this new client-go, as operator-util is meant to support all console enhancement like consoleyamlsamples, login page etc.